### PR TITLE
Feat: badge

### PR DESCRIPTION
**Pull request by thesudhanshupanda**

### CODE REVIEW

File: CSS/Components/badge.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+   max-width: 100%;
+   font-size: 1.8rem;
+   margin: 0 1rem;
+}
+
+.badge-number{
+    font-size: 1rem;
+    background-color: var(--primary-color);
+    border-radius: 50%;
+    /* color: var(--off-white); */
```
Comment (vaibhav-mahalle) [inline CSS/Components/badge.css]: remove commented code.

File: CSS/Components/badge.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    height: 25px;
+    width: 30px;
```
Comment (vaibhav-mahalle) [inline CSS/Components/badge.css:13]: try and use rems instead of pixels.

File: CSS/Components/badge.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    top: -4px;
+    background-color: var(--danger-color);
+  }
+
+  .badge-profile {
+    bottom: -1px;
+    right: 15px;
+    position: absolute;
+    height: 15px;
+    width: 15px;
```
Comment (vaibhav-mahalle) [inline CSS/Components/badge.css:34]: same as above.

File: docs/Badges/Badge.html [hunk 4]
```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Merakii UI | Badges </title>
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
```
Comment (vaibhav-mahalle) [inline docs/Badges/Badge.html:10]: you can import this in the main CSS file so you don't have to paste the link every time.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some changes. please look into them. These are minor changes so you can do them in refactor if you are short on time.

File: CSS/Components/badge.css (hunk 1 above)
Comment (thesudhanshupanda) [reply CSS/Components/badge.css]: alright thanks !

File: docs/Badges/Badge.html [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+                  </li>
+                  <li><a href="../cards/card.html">Cards</a></li>
+                  <li><a href="../images/image.html">Images</a></li>
+                  <li><a href="../inputs/input.html">Input</a></li>
+                  <li><a href="../navbar/navbar.html">Navbar</a></li>
+                  <li><a href="../lists/list.html">List</a></li>
+                  <li><a href="../rating/rating.html">Rating</a></li>
+                  <li><a href="../toast/toast.html">Toast</a></li>
+                  <li><a href="../typography/typography.html">Typography</a></li>
+                  <li><a href="../grid/grid.html">Grid</a></li>
```
Comment (krishanu2911) [inline docs/Badges/Badge.html]: just a suggestion to align them in a same manner

**Review by krishanu2911 — APPROVED** [review]
Just rectify the correction addressed by Vaibhav above nothing else correction required else seems good to go just added a small suggestion below

File: docs/Badges/Badge.html (hunk 5 above)
Comment (thesudhanshupanda) [reply docs/Badges/Badge.html]: ok sure will do

**Pull request merged**